public/css/bulma.css

```diff
@@ -6239,9 +6239,6 @@ label.panel-block {
   align-items: flex-end | flex-start | baseline;
   align-content: flex-end | flex-start; }
 
-.card-body {
-  background: #e5e5e5 !important; }
```

Comment:
Não acredito que esse !important passou batido por tanto tempo 
![Judging you](https://www.awesomelyluvvie.com/wp-content/uploads/2012/11/JudgingYou.gif)